### DISCUSSION TITLE
Backlinks

### DISCUSSION THREAD

**Discussion by latvia234** [vincode-io/Zavala#117] [Ideas]
Backlinks are not mentioned in the [Planned Features](https://github.com/vincode-io/Zavala/wiki/Planned-Features), so let's make a new discussion.

@kerim mentioned backlinks in this post: https://github.com/vincode-io/Zavala/discussions/56#discussioncomment-271971.

I also think that backlinks are really useful, however I think that we should also keep the mission statement in mind:

> I would kill for a good simple outliner that synced across iOS and Mac, and had a great simple UI on all three platforms.

How could we keep Zavala simple with backlinks?

A screenshot how backlinks or linked references look like in Athens Research:

![backlinks](https://user-images.githubusercontent.com/86680397/130620866-9b862919-5bd3-4b33-b917-a9fa48e18844.png)

In the screenshot above you can see that there are no backlinks under "Linked References". But Athens Research just shows the title "Linked References", even if there are no backlinks. 

- Could it be an option to just hide the title "Linked References" or "Backlinks", if there are no backlinks available?
- Could it be an option to only show backlinks under the outline if there are backlinks available? 

This way we can keep Zavala just a simple outliner, and more options like backlinks only show up when you have backlinks available.

Related: it would be great if backlinks could be created [with wikilinks](https://github.com/vincode-io/Zavala/discussions/56#discussioncomment-1227133).

**Comment by vincode-io** [comment]
We do support back links. Just link from one outline to another and backlinks will appear at the bottom of the linked outline.

**Reply by latvia234** [reply]
That's great! I did not know that. I searched cmd+f for "backlinks" in all the Wiki pages and Discussions but could not find anything on how to use them.

I'm only using Zavala on iOS right now, maybe this is why I could not find how to use them.

When writing this reply I searched on the following website, and there it is mentioned: https://zavala.vincode.io/features/

> Use the Link dialog to quickly create a link to another Outline.

But in the Zavala iOS app I cannot find the Link dialog.
